phoenix-core/src/it/java/org/apache/phoenix/end2end/UCFWithDisabledIndexWithDDLValidationIT.java

```diff
@@ -40,7 +40,7 @@ private static void initCluster() throws Exception {
         Map<String, String> props = Maps.newConcurrentMap();
         props.put(BaseScannerRegionObserverConstants.PHOENIX_MAX_LOOKBACK_AGE_CONF_KEY,
                 Integer.toString(60 * 60 * 1000));
-        props.put(QueryServices.DEFAULT_UPDATE_CACHE_FREQUENCY_ATRRIB, "ALWAYS");
+        props.put(QueryServices.DEFAULT_UPDATE_CACHE_FREQUENCY_ATRRIB, "NEVER");
```

Comment:
do we want to test with both `ALWAYS` and `NEVER`?

Reply:
@virajjasani With DDL timestamp validation, UCF should not be ALWAYS, so I think it is okay. WDYT?